src/core/constants/appConfig.ts

```diff
@@ -83,19 +83,15 @@ const appConfig = {
 	ROUTES: {
 		DEFAULT: "/",
 		DEMO: "/demo",
+		NL_DEMO: "/demo?dp=Esri&nl=true",
```

Comment:
What is this used for and have we implemented this yet?

Reply:
This is used to redirect user to the NL search, this is implemented but controlled via an env flag